selfdrive/debug/print_docs_diff.py

```diff
@@ -1,120 +1,82 @@
 #!/usr/bin/env python3
```

Comment:
I'd keep CI/GitHub specific stuff out of this PR. You should just give it an old path and a new path, and selfdrive_tests.yaml should pass that in.

Reply:
Reverted GH Action changes. Will try again using your test.

Reply:
It still downloads files